gprofiler/profilers/node.py

```diff
@@ -61,6 +62,7 @@ def _get_dest_inside_container(musl: bool, node_version: str) -> str:
 def _start_debugger(pid: int) -> None:
     # for windows: in shell node -e "process._debugProcess(PID)"
     os.kill(pid, signal.SIGUSR1)
+    sleep(2)
```

Comment:
Instead of a sleep here, I think it makes more sense to re-try the connection if it fails early